packages/css/src/components/Table/table.css

```
@@ -0,0 +1,92 @@
+.table-container {
+  overflow-x: auto;
+  width: 100%;
+}
+
+.table {
+  display: table;
+  width: 100%;
+  border-spacing: 0;
+  border-collapse: collapse;
+  border-radius: var(--table-border-radius);
+  text-align: left;
+}
+
+.table-head {
+  display: table-header-group;
+  font: var(--table-header-font);
+}
+
+.table-body {
+  display: table-row-group;
+  font: var(--table-cell-font);
+}
+
+.table-row {
+  display: table-row;
+  outline: 0;
+  color: inherit;
+  vertical-align: middle;
+  font: inherit;
+}
+
+.table-cell {
+  display: table-cell;
+  padding: var(--padding-2) var(--padding-3);
+  border-bottom-width: var(--table-border-width);
+  border-bottom-style: solid;
+  border-bottom-color: var(--table-border-color);
+  background: var(--table-background-color);
+  color: var(--table-text-color);
+  font: inherit;
+}
+
+.table-cell:hover {
+  background: var(--table-hover-background-color);
+}
+
+th.table-cell {
+  color: var(--table-header-color);
+}
+
+.table-cell a {
+  color: var(--table-action-color);
+  text-decoration: none;
+}
+
+.table-cell:active,
+.table-cell.active {
+  background: var(--table-active-background-color);
+}
+
+.table-cell:active a,
+.table-cell.active a {
+  color: var(--table-active-action-color);
+}
+
+.table-cell:disabled,
+.table-cell.disabled {
+  background: var(--table-disabled-background-color);
+  color: var(--table-disabled-text-color);
+}
+
+th.table-cell:disabled,
+th.table-cell.disabled {
+  color: var(--table-disabled-header-color);
+}
```

Can we change that to 
```suggestion
.table-head .table-cell:disabled,
.table-head .table-cell.disabled {
  color: var(--table-disabled-header-color);
}
```

Same for other places that use `th` 

Otherwise it would be necessary to use `<th>` to create a table 

_(Judging by the classes and `display` values it seems it should be possible to create a table using any elements)_

---

You should always use `th` in `thead` though.

---

In that case why did you specify classes like `.table-head` with `display: table-header-group` ? 
That's the default for `thead`

---

I thought it was to allow to add the class to a `<div class="table-head" />` - you wouldn't put `th` elements inside a div - I'd just use a table